svelte/vite.config.js

```diff
@@ -3,4 +3,7 @@ import { defineConfig } from 'vite';
 
 export default defineConfig({
 	plugins: [sveltekit()]
+	server:{
+		host: '127.0.0.1'
```

Comment:
I assume that here the host was defaulting to `localhost` ?

Does this fix the problem you had?

Reply:
Yes to both questions.